PyZAPI/pyzapi/routes.py

```diff
@@ -0,0 +1,10 @@
+def includeme(config):
+    config.add_static_view('static', 'static', cache_max_age=3600)
+    config.add_route('home', '/')
+
+    # relevant routes for the blog post
```

Comment:
So, if you’re doing this in a restful manner… It should look more like: I don’t entirely know the framework yet, so you may need to do some tweaking. It’s not the end of the world for this PR, we can change it later.

Single user resoruce = GET /users/{user_id}
Collection of users = GET /users/

Create: POST /users/
Update: PUT /users/{user_id}

Delete: DELETE /users/{user_id}